libseccomp/src/lib.rs

```diff
@@ -786,11 +1008,17 @@ impl ScmpFilterContext {
         Ok(())
     }
 
-    /// export_bpf outputs Berkeley Packet Filter-formatted, kernel-readable dump of a
+    /// Outputs Berkeley Packet Filter-formatted, kernel-readable dump of a
```

Comment:
export_pfc uses the abrevation (PFC) while export_bpf uses the long form (Berkeley Packet Filter).

I'm not sure which one is better, but this is a bit inconsistent.

Reply:
I use `PFC(Pseudo Filter Code)-formatted` and `BPF(Berkeley Packet Filter)-formatted` for the consistent.